tests/unit/test-cmd/test.build.js

```diff
@@ -104,7 +104,7 @@ describe('build', () => {
             assert.instanceOf(error, UsageError);
             assert.match(
               error.message,
-              /Error .* file .*messages\.json: SyntaxError: Unexpected string/);
+              /Unexpected token '"' .* in .*messages.json/);
```

Comment:
How about splitting this assertion for readability, e.g.:

```js
assert.match(error.message, /Unexpected token '"' at 1:15/);
assert.include(error.message, messageFileName);
```

also, we should test the same change in "tests/unit/test-util/test.manifest.js", e.g. something like:
```diff
diff --git a/tests/unit/test-util/test.manifest.js b/tests/unit/test-util/test.manifest.js
index 166a276..18f8a95 100644
--- a/tests/unit/test-util/test.manifest.js
+++ b/tests/unit/test-util/test.manifest.js
@@ -70,7 +70,8 @@ describe('util/manifest', () => {
           .then(() => getValidatedManifest(tmpDir.path()))
           .then(makeSureItFails())
           .catch(onlyInstancesOf(InvalidManifest, (error) => {
-            assert.match(error.message, /Error parsing manifest\.json/);
+            assert.match(error.message, /Error parsing manifest\.json at /);
+            assert.include(error.message, 'Unexpected token \' \' at 2:49');
             assert.include(error.message, manifestFile);
           }));
       }
```